modules/migration-mtc-support.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/about-mtc.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-support_{context}"]
+= {mtc-short} 1.8 support
+
+* {mtc-short} 1.8.3 and earlier using {oadp-short} 1.3.z is supported on all {OCP-short} versions 4.15 and earlier.
+
+* {mtc-short} 1.8.4 and later using {oadp-short} 1.3.z is currently supported on all {OCP-short} versions 4.15 and earlier.
+
+* {mtc-short} 1.8.4 and later using {oadp-short} 1.4.z is currently supported on all supported {OCP-short} versions 4.13 and later.
```

Comment:
Is there a difference between _supported_ in the first bullet and _currently supported_ in the second two?

Reply:
Yeah 1.8.4 features a major dependency update (there was lots of discussion about if it would be 1.9.0

So we want to provide users the information about the earlier versions.

Reply:
OMG ... i am a deeply boring man